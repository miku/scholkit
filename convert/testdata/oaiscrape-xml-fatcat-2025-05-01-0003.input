<?xml version="1.0"?>
<record xmlns="http://www.openarchives.org/OAI/2.0/">
  <header status="">
    <identifier>oai:ojs.pkp.sfu.ca:article/1371</identifier>
    <datestamp>2021-10-08T16:27:15Z</datestamp>
    <setSpec>jecs:EXP</setSpec>
    <setSpec>driver</setSpec>
  </header>
  <metadata>
    <oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
      <dc:title xml:lang="en-US">Organisational foundations of functioning of the Right-Bank Ukraine orthodox brotherhoods from the 1850s to 1900s</dc:title>
      <dc:creator>Aloshyna, Oksana </dc:creator>
      <dc:creator>Smyrnov, Andrii</dc:creator>
      <dc:creator>Yankovska, Zhanna </dc:creator>
      <dc:creator>Blyzniak, Mykola</dc:creator>
      <dc:creator>Marchuk, Volodymyr </dc:creator>
      <dc:subject xml:lang="en-US">Russian government</dc:subject>
      <dc:subject xml:lang="en-US">Orthodox fraternities</dc:subject>
      <dc:subject xml:lang="en-US">organizational foundations</dc:subject>
      <dc:subject xml:lang="en-US">charter</dc:subject>
      <dc:description xml:lang="en-US">Aim. This research aims to reveal the peculiarities of standardization and organizational principles behind the functioning of Orthodox brotherhoods on the territory of Right-Bank Ukraine from the 1850s to the 1900s.
Methods. Methodologically, the authors of the work rely on the principles of novelty, objectivity, and historicism and employ general scientific methods (internal criticism of the sources, analysis, synthesis, generalization).
Results. The study revealed that during the second half of the 19th century, the activity of Orthodox brotherhoods on the territories controlled by the Russian Empire was regulated by the law &#x201C;Basic Rules for the Establishment of the Orthodox Church Brotherhoods&#x201D; which regulated the prioritized tasks, membership and main vectors of their work. At the beginning of the 20th century, some changes occurred in the social-political life of the Russian Empire, which also affected the position of the Orthodox brotherhoods of Right-Bank Ukraine.
Conclusion. The church authorities devoted considerable efforts to revitalizing and restoring the activity of the Orthodox brotherhoods at the beginning of the 20th century. In order to find new solutions to the situation, they discussed the further functioning of the fraternities at the congresses in which participated the representatives of the Orthodox brotherhoods of the western provinces of the Russian Empire. Thus, the church management controlled brotherhoods and channelled them into the required course of action.</dc:description>
      <dc:publisher xml:lang="en-US">Foundation Pro Scientia Publica/ University of Wroc&#x142;aw</dc:publisher>
      <dc:date>2021-09-25</dc:date>
      <dc:type>info:eu-repo/semantics/article</dc:type>
      <dc:type>info:eu-repo/semantics/publishedVersion</dc:type>
      <dc:type xml:lang="en-US">text</dc:type>
      <dc:format>application/pdf</dc:format>
      <dc:identifier>https://e-methodology.eu/index.php/jecs/article/view/1371</dc:identifier>
      <dc:identifier>10.15503/jecs2021.2.242.250</dc:identifier>
      <dc:source xml:lang="en-US">Journal of Education Culture and Society; Vol. 12 No. 2 (2021): Journal of Education Culture and Society; 242-250</dc:source>
      <dc:source>2081-1640</dc:source>
      <dc:source>2081-1640</dc:source>
      <dc:language>eng</dc:language>
      <dc:relation>https://e-methodology.eu/index.php/jecs/article/view/1371/1165</dc:relation>
      <dc:rights xml:lang="en-US">Copyright (c) 2021 Andrii Smyrnov, Oksana  Aloshyna, Zhanna  Yankovska, Mykola Blyzniak, Volodymyr  Marchuk</dc:rights>
      <dc:rights xml:lang="en-US">http://creativecommons.org/licenses/by/4.0</dc:rights>
    </oai_dc:dc>
  </metadata>
  <about/>
</record>
