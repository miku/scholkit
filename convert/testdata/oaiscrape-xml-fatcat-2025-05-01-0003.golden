{
    "source": "oaiscrape",
    "contribs": [
        {
            "extra": {},
            "raw_name": "Aloshyna, Oksana"
        },
        {
            "extra": {},
            "raw_name": "Smyrnov, Andrii"
        },
        {
            "extra": {},
            "raw_name": "Yankovska, Zhanna"
        },
        {
            "extra": {},
            "raw_name": "Blyzniak, Mykola"
        },
        {
            "extra": {},
            "raw_name": "Marchuk, Volodymyr"
        }
    ],
    "ext_ids": {
        "doi": "10.15503/jecs2021.2.242.250",
        "oai": "oai:ojs.pkp.sfu.ca:article/1371"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {
            "set_spec": [
                "jecs:EXP",
                "driver"
            ],
            "url": [
                "https://e-methodology.eu/index.php/jecs/article/view/1371"
            ]
        }
    },
    "id": "oaiscrape-6c468254df07161fe8edcd8494dbc87fab3a6ea6",
    "language": "eng",
    "release_date": "2021-09-25",
    "title": "Organisational foundations of functioning of the Right-Bank Ukraine orthodox brotherhoods from the 1850s to 1900s"
}
